tensorboard/components/vz_chart_helpers/vz-chart-helpers.ts

```diff
@@ -102,7 +102,7 @@ export interface TooltipColumn {
   title: string;
   // This function computes the value for the string. None of the arguments
   // passed to it are ever falsy.
```

Comment:
Now that it's `status?` that argument can be falsy, so this needs to be updated, right?